src/css/styles.scss

```diff
@@ -187,3 +188,31 @@ ul {
   color: white;
 }
 
+.grid-item:hover {
+  border-color: hsla(0, 0%, 100%, 0.5);
+  cursor: move;
+}
+
+.friend-icon-container {
+  height: 100%;
+  background: #ffffff;
+  padding: 5% 5%;
```

Comment:
you can just make this 5% - same as saying 5% 5% and just cleans it up a bit.

Reply:
No problem, done